test/pummel/test-process-cpuUsage.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+const start = process.cpuUsage();
+
+// process.cpuUsage() should return {user: aNumber, system: aNumber}
+assert(start != null);
+assert(Number.isFinite(start.user));
+assert(Number.isFinite(start.system));
+
+// Run a busy-loop for 2 seconds.
+const now = Date.now();
+while (Date.now() - now < 2000);
```

Comment:
Let's drop the time on this to 500ms. 2 seconds is a long time for a test to run. Generally we like tests to run in < 100ms.